### PR TITLE
DBZ-7594 Correct the negative value of data type TIME replicated from MySQL.

### PR DESCRIPTION
**Pull request by VWagen1989**
This PR addresses the issue [DBZ-7594](https://issues.redhat.com/browse/DBZ-7594).

### CODE REVIEW

File: debezium-connector-mysql/src/main/java/io/debezium/connector/mysql/MySqlValueConverters.java [hunk 1]
```diff
@@ -29,35 +54,8 @@
...
-import io.debezium.jdbc.JdbcValueConverters;
-import io.debezium.jdbc.TemporalPrecisionMode;
-import io.debezium.relational.Column;
-import io.debezium.relational.Table;
-import io.debezium.relational.ValueConverter;
-import io.debezium.time.Year;
-import io.debezium.util.Loggings;
-import io.debezium.util.Strings;
+import static io.debezium.config.CommonConnectorConfig.EventConvertingFailureHandlingMode.FAIL;
+import static io.debezium.config.CommonConnectorConfig.EventConvertingFailureHandlingMode.WARN;
```
Comment (Naros) [inline debezium-connector-mysql/src/main/java/io/debezium/connector/mysql/MySqlValueConverters.java:58]: Please do not reorder the imports, use `./mvnw clean install -DskipITs -DskipTests` to apply the code style to move these back to the proper order and squash the change.  Thanks.

File: debezium-connector-mysql/src/main/java/io/debezium/connector/mysql/RowDeserializers.java [hunk 2]
```diff
@@ -5,33 +5,22 @@
...
 import io.debezium.DebeziumException;
 import io.debezium.config.CommonConnectorConfig.EventProcessingFailureHandlingMode;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.IOException;
+import java.io.Serializable;
+import java.time.*;
+import java.util.Calendar;
+import java.util.Map;
```
Comment (Naros) [inline debezium-connector-mysql/src/main/java/io/debezium/connector/mysql/RowDeserializers.java:23]: Same as above.

**Review by Naros — CHANGES_REQUESTED** [review]
Hi, just a few comments regarding code style.

File: debezium-connector-mysql/src/main/java/io/debezium/connector/mysql/MySqlValueConverters.java (hunk 1 above)
Comment (VWagen1989) [reply debezium-connector-mysql/src/main/java/io/debezium/connector/mysql/MySqlValueConverters.java]: Thanks for your feedbacks! Done.

File: debezium-connector-mysql/src/main/java/io/debezium/connector/mysql/RowDeserializers.java (hunk 2 above)
Comment (VWagen1989) [reply debezium-connector-mysql/src/main/java/io/debezium/connector/mysql/RowDeserializers.java]: Thanks for your feedbacks! Done.

**Comment by Naros** [comment]
Hi @VWagen1989, it would seem the test failures are related.  Could you take a look?

**Comment by VWagen1989** [comment]
> Hi @VWagen1989, it would seem the test failures are related. Could you take a look?

OK, I've solved this problem and rerun the failed cases locally, and they were all passed.

**Comment by jpechane** [comment]
@VWagen1989 Thank you very much. I triggered a rebuild. Could you please also add a test? Ideeally it should test both snapshot and streaming acquired values for both scenarios that were mentioned in the original Jira report.

**Comment by VWagen1989** [comment]
> @VWagen1989 Thank you very much. I triggered a rebuild. Could you please also add a test? Ideeally it should test both snapshot and streaming acquired values for both scenarios that were mentioned in the original Jira report.

@jpechane Hi, I've fix the new problem described in the comment of [DBZ-7594](https://issues.redhat.com/browse/DBZ-7594?focusedId=24289825&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-24289825). And the corresponding test cases are also added. Please take a look.

**Comment by VWagen1989** [comment]
@jpechane @Naros Hi, I found this error in the log of the test. It seems something wrong with the image `quay.io/debezium/connect:nightly`?
```
Error:  Errors: 
Error:    DebeziumContainerTest » Completion org.testcontainers.containers.ContainerLaunchException: Container startup failed for image quay.io/debezium/connect:nightly
```

**Comment by jpechane** [comment]
@VWagen1989 I restarted the MariaDB job, it might be a network glitch.

**Review by Naros — APPROVED** [review]

**Comment by jpechane** [comment]
@VWagen1989 Applied, thanks a lot!

**Pull request merged**